label_maker/main.py

```diff
@@ -18,17 +19,20 @@
 
 logger = logging.getLogger(__name__)
 
+
 def parse_args(args):
     """Create an argument parser with subcommands"""
     desc = 'label_maker (v%s)' % __version__
     dhf = argparse.ArgumentDefaultsHelpFormatter
     parser = argparse.ArgumentParser(description=desc)
+    config_fpath = op.join(op.dirname(__file__), '..', 'config.json')
```

Comment:
@wronk can we leave this one as is? The default config should be relative to the users directory and not the module